keras/engine/base_layer.py

```diff
@@ -226,7 +226,8 @@ def add_weight(self,
                    initializer=None,
                    regularizer=None,
                    trainable=True,
-                   constraint=None):
+                   constraint=None,
+                   sparse_weight=False):
```

Comment:
Add explanation in doc string, note only used for mxnet

Reply:
added